weave-js/src/components/PagePanelComponents/Home/Browse3/inference/ModelDetailsLoaded.tsx

```diff
@@ -0,0 +1,331 @@
+  Python: 'python',
+  TypeScript: 'typescript',
+  Curl: 'shell',
+};
+
+export const ModelDetailsLoaded = ({
+  model,
+  inferenceContext,
+}: ModelDetailsLoadedProps) => {
+  const [selectedLanguage, setSelectedLanguage] = useState('Python');
+
+  const label = getModelLabel(model);
+  const logo = getModelLogo(model);
+
+  const history = useHistory();
+  const onOpenPlayground = () => {
+    navigateToPlayground(history, model.id, inferenceContext);
+  };
+  const hasPlayground = !!model.idPlayground && inferenceContext.isLoggedIn;
+  const tooltipPlayground = hasPlayground
+    ? undefined
+    : inferenceContext.isLoggedIn
+    ? 'This model is not available in the playground'
+    : 'You must be logged in to use the Playground';
+
+  const codeExamples =
+    model.apiStyle === 'embedding'
+      ? CODE_EXAMPLES_EMBEDDING
+      : CODE_EXAMPLES_CHAT;
+  let codeExample = codeExamples[selectedLanguage] ?? '';
+  codeExample = codeExample.trim();
+  codeExample = codeExample.replace('{model_id}', model.idPlayground);
+
+  const onClickCopy = useCallback(() => {
+    copyToClipboard(codeExample);
+    toast.success('Copied to clipboard');
+  }, [codeExample]);
+
+  return (
+    <div className="px-32 py-24">
+      <div className="flex items-start">
+        <div className="flex flex-grow flex-col">
+          <div className="text-sm font-semibold">
+            <Link to={INFERENCE_PATH} className="flex items-center gap-2">
+              <Icon width={16} height={16} name="back" /> Inference
+            </Link>
+          </div>
+          <div>
+            <div className="flex items-center gap-8">
+              <div className="text-2xl">{label}</div>
+              {model.modalities && <Modalities modalities={model.modalities} />}
+            </div>
+          </div>
+        </div>
+        <div className="flex items-center gap-8">
+          {/* <Button size="large" variant="secondary">
+            Evaluate model
+          </Button> */}
+          <Button
+            size="large"
+            onClick={onOpenPlayground}
+            disabled={!hasPlayground}
+            tooltip={tooltipPlayground}>
+            Try in Playground
+          </Button>
+        </div>
+      </div>
+      <div className="mb-8 mt-16 font-semibold">Model overview</div>
+
+      <div className="mb-16 flex gap-10">
+        <Tooltip
+          trigger={
+            <DetailTile header="Price" footer="Input - Output">
+              <div className="text-xl font-semibold">
+                {getPriceString(model, false, '-')}
+              </div>
+            </DetailTile>
+          }
+          content="Price per million tokens"
+        />
+        {model.contextWindow && (
+          <DetailTile header="Context window">
+            <div className="text-xl font-semibold">
+              {getContextWindowString(model)}
+            </div>
+          </DetailTile>
+        )}
+        {/* <DetailTile header="Parameters" /> */}
```

Comment:
same here, i would feel better if these had TODO or some way of finding them easily in the future, presumably as fast follows implement them